pyEpiabm/pyEpiabm/toy_population_config.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+from .population import Population
+from .household import Household
+
+
+class ToyPopulation:
+    """ Class that creates a toy population for use and returns in the simple
+    python model.
+    """
+    def make_pop(self, population_size: int, cell_number: int,
+                 microcell_per_cell: int, household_number: int,
+                 if_households: bool = False):
+        """Method that initializes a population class with a given population size,
+        number of cells and number of microcells. A multinomial distribution is
+        used to distribute the number of people into the different microcells.
+        There is also an option to distribute people into households.
+
+        : param pop_size: Total number of people in population.
+        : type pop_size: int
+        : param cell_number: Number of cell objects the population will be
+          split in to.
+        : type cell_number: int
+        : param microcell_per_cell: Number of microcell objects per cell.
+        : type microcell_per_cell: int
+        : param household_number: Number of households per microcell.
+        : type household_number: int
+        : param if_households: decides whether to put people into households.
+        : type if_households: bool
+
+        : return new_pop: Population
+        : rtype new_pop: `Population` class object
+        """
+        # Initialise a population class
+        new_pop = Population()
+
+        # Checks parameter type and stores as class objects
+        if not isinstance(if_households, bool):
+            raise TypeError('Include household input needs to be boolean')
+        self.pop_size = population_size
+        self.total_number_microcells = cell_number*microcell_per_cell
+        self.if_households = if_households
+        self.household_number = household_number
```

Comment:
Why are we storing these internal variables? If we want these numbers to be stored they should be stored within the Population rather than ToyPopulation. Alternatively they could be returned by make_population